loader/loader.go

```diff
@@ -44,6 +45,35 @@ func NewLoader() ObjectLoader {
 	return &loader{}
 }
 
+// GetActiveManagedReleasePipelineRuns returns all active managed Release PipelineRuns for the Application being Released.
+// PipelineRuns for the Release passed as an argument are ignored.
+func (l *loader) GetActiveManagedReleasePipelineRuns(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*tektonv1.PipelineRunList, error) {
```

Comment:
Can we use unfinished or something instead of active? We already have active for the RP/RPA stuff